src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -118,8 +118,9 @@ public function cgetAction(Request $request)
         $locale = $this->getRequestParameter($request, 'locale', true);
         $fieldDescriptors = $this->getFieldDescriptors($locale, false);
         $ids = array_filter(explode(',', $request->get('ids')));
+        $excludedIds = array_filter(explode(',', $request->get('excluded-ids')));
```

Comment:
I have just realized that we are inconsistent in the naming here... Sometimes we use a dash and sometimes we use camelCase for these parameters... And in the action for loading the content of the smart content it is just called `excluded`... Maybe go with that, in order to avoid that issue 🙈